covalent/_file_transfer/strategies/s3_strategy.py

```diff
@@ -0,0 +1,68 @@
+import os
+import urllib.request
+from pathlib import Path
+from subprocess import PIPE, CalledProcessError, Popen
+from typing import Optional
+
+from covalent._file_transfer import File
+from covalent._file_transfer.strategies.transfer_strategy_base import FileTransferStrategy
+
+import boto3
```

Comment:
@poojithurao  this needs to be added in reqs. 

@wjcunningham7 , can we make this in a way like `pip install covalent['aws']` adds this as additional requirements  and then in future tag along executers as well ?

Reply:
@santoshkumarradha I have added boto3 to the requirements.txt